test/suites/snapshot.robot

```diff
@@ -41,14 +41,24 @@ Snapshotter Smoke Test
 
 *** Keywords ***
 Test Suite Setup
-    [Documentation]    Setup test namespace, patch the lvmd for thin-volume support, and restart microshift
+    [Documentation]    Setup test namespace, patch the lvmd for thin-volume support, and restart microshift for
```

Comment:
Could you remove the `robot:skip-on-failure` tag on line 30 as part of this PR to ensure we're landing the job in a green state? I see it's passing, so should be OK.